apps/sms/views/conversation/js/conversation.js

```diff
@@ -2537,7 +2582,7 @@ var ConversationView = {
       );
     }
 
-    this.emit('recipientschange');
+    Compose.refresh();
```

Comment:
I think we should use a throttled version of toFieldInput (so both for listContacts and Compose.refresh) -- not for this patch but we could file a separate mentored bug